hal/network/lwip/wiznet/wiznetif.h

```diff
@@ -83,10 +86,9 @@ class WizNetif : public BaseNetif {
 
 private:
     hal_spi_interface_t spi_;
-    hal_pin_t cs_;
-    hal_pin_t reset_;
-    hal_pin_t interrupt_;
-    bool postpone_;
+    hal_pin_t cs_ = PIN_INVALID;
+    hal_pin_t reset_ = PIN_INVALID;
+    hal_pin_t interrupt_ = PIN_INVALID;
```

Comment:
Use defaults?
HAL_PLATFORM_ETHERNET_WIZNETIF_CS_PIN_DEFAULT
HAL_PLATFORM_ETHERNET_WIZNETIF_RESET_PIN_DEFAULT
HAL_PLATFORM_ETHERNET_WIZNETIF_INT_PIN_DEFAULT

Reply:
The defaults are in the WizNetConfig. I've left things as much as possible to how it was before: settings coming from `network.cpp`. Could set to defaults here too.